ts/components/messages/MessageAgenda.tsx

```diff
@@ -78,8 +78,9 @@ const styles = StyleSheet.create({
   },
   sectionHeaderText: {
     fontSize: 18,
-    lineHeight: 20,
-    color: customVariables.brandDarkestGray
+    color: customVariables.brandDarkestGray,
+    ...makeFontStyleObject(Platform.select, "600", false),
```

Comment:
`makeFontStyleObject` has isItalic default value at false
```suggestion
    ...makeFontStyleObject(Platform.select, "600"),
```

Reply:
Fixed within https://github.com/teamdigitale/io-app/pull/1280/commits/65755be70601628e243b982dde5b7e37f572e649